[2022-02-12 20:52:29,851] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.create_awards 2022-02-12T20:51:50.361694+00:00 [queued]>
[2022-02-12 20:52:29,962] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.create_awards 2022-02-12T20:51:50.361694+00:00 [queued]>
[2022-02-12 20:52:29,976] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 20:52:29,976] {taskinstance.py:881} INFO - Starting attempt 1 of 2
[2022-02-12 20:52:29,977] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 20:52:30,014] {taskinstance.py:901} INFO - Executing <Task(PostgresOperator): create_awards> on 2022-02-12T20:51:50.361694+00:00
[2022-02-12 20:52:30,020] {standard_task_runner.py:54} INFO - Started process 4503 to run task
[2022-02-12 20:52:30,566] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', '1_load_initial_data', 'create_awards', '2022-02-12T20:51:50.361694+00:00', '--job_id', '22', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/load_data.py', '--cfg_path', '/tmp/tmpuq9rzeyr']
[2022-02-12 20:52:30,570] {standard_task_runner.py:78} INFO - Job 22: Subtask create_awards
[2022-02-12 20:52:30,755] {logging_mixin.py:112} INFO - Running <TaskInstance: 1_load_initial_data.create_awards 2022-02-12T20:51:50.361694+00:00 [running]> on host f261333f79b3
[2022-02-12 20:52:30,828] {postgres_operator.py:62} INFO - Executing: 
                      CREATE TABLE IF NOT EXISTS 
                      dbt_raw_data.awards 
                      (taxid integer, date varchar(20), name varchar(100));
                      
[2022-02-12 20:52:30,866] {base_hook.py:89} INFO - Using connection to: id: dbt_postgres_instance_raw_data. Host: postgres-dbt, Port: 5432, Schema: dbtdb, Login: dbtuser, Password: XXXXXXXX, extra: None
[2022-02-12 20:52:30,897] {dbapi_hook.py:176} INFO - 
                      CREATE TABLE IF NOT EXISTS 
                      dbt_raw_data.awards 
                      (taxid integer, date varchar(20), name varchar(100));
                      
[2022-02-12 20:52:30,901] {postgres_operator.py:67} INFO - NOTICE:  relation "awards" already exists, skipping

[2022-02-12 20:52:30,994] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=1_load_initial_data, task_id=create_awards, execution_date=20220212T205150, start_date=20220212T205229, end_date=20220212T205230
[2022-02-12 20:52:34,799] {local_task_job.py:102} INFO - Task exited with return code 0
